publish.js

```diff
@@ -0,0 +1,293 @@
+#! /usr/bin/env node
+
+/*
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var publish = {};
+var path = require("path");
+var pkgPath = path.join(__dirname, "package.json");
+var pkg = require(pkgPath);
+var extend = require("extend");
+
+// The package.json file of the top level package which is
+// running this module.
+var modulePkgPath = path.join(process.cwd(), "package.json");
```

Comment:
Actually I think it would be even better if this went into "options", and just defaulted to process.cwd() - otherwise we have a global dependency which prevents this package from being used by multiple clients at the same time